coretex/cli/modules/node.py

```diff
@@ -105,6 +109,44 @@ def registerNode(name: str) -> str:
     return accessToken
 
 
+def selectModelId(retryCount: int = 0) -> int:
+    if retryCount >= 3:
+        raise RuntimeError("Failed to fetch Coretex Model. Terminating...")
+
+    modelId = clickPrompt("Specify Coretex Model ID that you want to use:", type = int)
+
+    if not isinstance(modelId, int):
```

Comment:
In `clickPrompt` type was already marked as `int`, is there a need to validate it here again?